test/configs/zombieTanssi.json

```diff
@@ -30,47 +30,55 @@
     "parachains": [
         {
             "id": 1000,
-            "chain_spec_path": "specs/tanssi-1000.json",
+            "chain_spec_command": "../target/release/tanssi-node build-spec --chain dancebox-local --parachain-id 1000 --add-container-chain specs/template-container-2000.json --add-container-chain specs/template-container-2001.json --invulnerable 'Collator1000-01' --invulnerable 'Collator1000-02' --invulnerable 'Collator2000-01' --invulnerable 'Collator2000-02' --invulnerable 'Collator2001-01' --invulnerable 'Collator2001-02' --invulnerable 'Collator2002-01' --invulnerable 'Collator2002-02' > specs/tanssi-1000.json",
             "COMMENT": "Important: these collators will not be injected to pallet-invulnerables because zombienet does not support that. When changing the collators list, make sure to update `scripts/build-spec.sh`",
             "collators": [
                 {
                     "name": "Collator1000-01",
                     "ws_port": "9948",
-                    "command": "../target/release/tanssi-node"
+                    "command": "../target/release/tanssi-node",
+                    "args": ["-lparachain=debug"]
```

Comment:
This shouldn't be needed because it's defined as part of default_args above. Maybe you meant this, but not sure if it does anything

```suggestion
                    "args": ["-- -lparachain=debug"]
```

Reply:
yeah I am going to remove it